R/log_lik.R

```diff
@@ -456,8 +455,8 @@ ll_args.stanreg <- function(object, newdata = NULL, offset = NULL, m = NULL,
 .ll_polr_i <- function(data_i, draws) {
   eta <- linear_predictor(draws$beta, .xdata(data_i), data_i$offset)
   f <- draws$f
-  J <- draws$max_y
   y_i <- data_i$y
+  J <- ncol(draws$zeta) + 1
```

Comment:
Don't use `max_y` since `max_y` can change if using `newdata`.

Reply:
Also, Andrew made us put in support for estimating more categories than there are in the original outcome anyway.